src/Microsoft.AspNetCore.OData.Authorization/Extensions/ODataBuilderExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.AspNet.OData.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.AspNetCore.OData.Authorization.Extensions
+{
+    public static class ODataBuilderExtensions
+    {
+        public static IODataBuilder AddAuthorization(this IODataBuilder odataBuilder, Action<ODataAuthorizationOptions> configureODataAuth = null)
```

Comment:
>IODataBuilder odataBuilder [](start = 58, length = 26)

Do we use this "IODataBuilder" interface?

Reply:
That's what's returned by `services.AddOData()`. I added this extension to allow the developer to write `services.AddOData().AddAuthorization()`.